src/components/home/Home.js

```diff
@@ -57,6 +57,10 @@ function Home() {
       ? null
       : [...new Set(collections?.map(({ creator }) => creator).toArray())],
   );
+  const collectionsGraasper = collections.filter(
+    (collection) =>
+      collection.creator == 'e890a236-3d45-4042-8a95-c91120f4db73',
```

Comment:
Could you use an env variable? Like we do for auth_host, etc.

Reply:
Added the creator id for Graasper in .env.local:
GRAASPER_ID=e890a236-3d45-4042-8a95-c91120f4db73